aten/src/ATen/native/Interpolate.cpp

```diff
@@ -0,0 +1,109 @@
+#include <ATen/ATen.h>
```

Comment:
the name of this file might be confusing, given that we have a `interpolate` function which does upsampling / downsampling, and those functions are going to be ported to ATen soon

Reply:
Thank you for mentioning this, I forgot about those functions.